src/assets/scripts/airport/AirportController.js

```diff
@@ -188,4 +188,19 @@ export default class AirportController {
     hasAirport(icao) {
         return _has(this.airport.airports, icao);
     }
+
+    /**
+     * Remove an aircraft from the queue of any runway(s) at the AirportModel
+     * @for AirportModel
+     * @method removeAircraftFromAllRunwayQueues
+     * @param  {aircraft} aircraft The aircraft to remove
+     */
+    removeAircraftFromAllRunwayQueues(aircraft) {
+        const runways = this.airport_get().runways;
+        for (let runwayPair = 0; runwayPair < runways.length; runwayPair++) {
+            runways[runwayPair][0].removeQueue(aircraft, true);
+            runways[runwayPair][1].removeQueue(aircraft, true);
```

Comment:
I love what you did with the for iterator, that adds more context and makes it a little easier to read.  But, what are the `[0]` and `[1]` indicies for and also what does the `true` mean?

_sidenote:_ I would love to refactor how the `RunwayModel` is structured/used someday soon.  Things like the double indicies above should not be needed.